framework/platform/bpl/uci/cfg/bpl_cfg.cpp

```diff
@@ -179,7 +179,9 @@ int cfg_get_wifi_params(const char iface[BPL_IFNAME_LEN], struct BPL_WLAN_PARAMS
     // fail when wifi still works fine, so default to "auto" (0) and if
     // can't get the channel from UCI just move on.
     wlan_params->channel = 0;
-    cfg_get_channel(index, &wlan_params->channel);
+    if (!cfg_get_channel(index, &wlan_params->channel)) {
+        MAPF_INFO("UCI: radio" << index << ": channel is not configured assuming auto\n");
```

Comment:
I don't think the `\n` is needed.

Reply:
I had the same hunch, but given that all other messages there are \n terminated decided that if not for correctness then for uniformity it has to be there :-)